app/code/community/Oyst/OneClick/etc/system.xml

```diff
@@ -652,6 +654,29 @@
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
                         </position>
+
+                        <!-- class -->
+                        <class translate="label">
+                            <label>Add custom class to button</label>
+                            <frontend_type>text</frontend_type>
+                            <config_path>oyst/oneclick/checkout_cart_button_class</config_path>
+                            <sort_order>1</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                        </class>
+
+                        <!-- styles -->
+                        <styles translate="label comment">
```

Comment:
remove this method, duplicate with https://github.com/oystparis/oyst-1click-magento/pull/252/files